### PR TITLE
New form

### PR DESCRIPTION
**Pull request by pettinarip**
#7

### CODE REVIEW

**Comment by samajammin** [comment]
@pettinarip general design note... we should set a max width on this form container.

Here's my 13" mac. Looks awkard to me:
Screen Shot 2022-06-09 at 5 38 15 PM

Check out the old ESP site designs for comparison but I think we set a max width & centered the form container.

File: src/pages/apply.js [hunk 1]
```diff
@@ -0,0 +1,1066 @@
...
+        </span>
+        <div>
+          <small>(max 300 words)</small>
+        </div>
+        <Input
+          type="text"
+          name="projectLeaderReasons"
+          value={formState.projectLeaderReasons?.value}
+          onChange={handleInputChange}
+          maxLength="255"
```
Comment (samajammin) [inline src/pages/apply.js:750]: 255 character limit for 300 words? 🤔

Comment (pettinarip) [reply src/pages/apply.js:750]: Uhm yes, I copy pasted the 300 words from the spreadsheet. Should be 255 in the help text. Will confirm this with Sara

**Comment by samajammin** [comment]
Another small styling issue @pettinarip:
Screen Shot 2022-06-17 at 8 40 38 AM

Reminder to address my comment here as well: https://github.com/ethereum/fellowship-program-website/pull/8#issuecomment-1151650687

File: src/pages/apply.js [hunk 2]
```diff
@@ -0,0 +1,1169 @@
...
+
+const StyledButton = styled(Button)`
+  margin-bottom: 1rem;
+`
+
+const GENDER_TYPES = ["She/Her", "Him/His", "They/Them", "Other"]
+
+const YESNO = ["Yes", "No"]
+
+const PROJECTRESEARCHIDEA = [
```
Comment (samajammin) [inline src/pages/apply.js:59]:
```suggestion
const PROJECT_RESEARCH_IDEA = [
```
Let's continue the format of separating words with underscore. Could also be e.g. `PROJECT_IDEA`.

File: src/pages/apply.js [hunk 3]
```diff
@@ -0,0 +1,1169 @@
...
+  "Yes, I have a project/research being implemented",
+  "Yes, I have a project idea but it is in its early stages",
+  "No, I have an idea, but the project/research is not currently being implemented",
+  "Other",
+]
+
+const REFERRALSOURCE = [
```
Comment (samajammin) [inline src/pages/apply.js:66]:
```suggestion
const REFERRAL_SOURCE = [
```
Same here.

File: src/pages/apply.js [hunk 4]
```diff
@@ -0,0 +1,1169 @@
...
+  "memeLink",
+]
+
+const RequiredError = () => <ErrorMessage>Field is required</ErrorMessage>
+const EmailError = () => (
+  <ErrorMessage>Please provide a valid email address</ErrorMessage>
+)
+const UrlError = () => <ErrorMessage>Ensure link is a valid URL</ErrorMessage>
+
+const DevconGrantsForm = () => {
```
Comment (samajammin) [inline src/pages/apply.js:124]:
```suggestion
const ApplicationForm = () => {
```
Let's rename this component 😏

File: src/pages/apply.js [hunk 5]
```diff
@@ -0,0 +1,1169 @@
...
+            {formState.phone.isTouched && !formState.phone.isValid && (
+              <RequiredError />
+            )}
+          </ErrorDiv>
+        </StyledLabel>
+        <StyledLabel>
+          <span>
+            Which gender pronoun do you use? <Required>*</Required>
+          </span>
+          <StyledSelect
```
Comment (samajammin) [inline src/pages/apply.js:506]: This field should probably be just a dropdown select vs. having search functionality.

File: src/pages/apply.js [hunk 6]
```diff
@@ -0,0 +1,1169 @@
...
+            {formState.title.isTouched && !formState.title.isValid && (
+              <RequiredError />
+            )}
+          </ErrorDiv>
+        </StyledLabel>
+        <StyledLabel>
+          <span>
+            Are you affiliated with any organization(s)? <Required>*</Required>
+          </span>
+          <StyledSelect
```
Comment (samajammin) [inline src/pages/apply.js:600]: This yes/no question should definitely not be a search select 😄 - maybe radio buttons?

**Review by samajammin — COMMENTED** [review]
👍

**Review by samajammin — APPROVED** [review]

File: src/pages/index.js [hunk 7]
```diff
@@ -228,76 +229,92 @@ class IndexPage extends React.Component {
...
+                <Box>
+                  <p className="title">
+                    Now accepting applications to the 2022 Fellowship Program
+                  </p>
+                  <p className="teaser">
+                      If you are working on a project that can help Ethereum reach
+                      the next billion users,
+                    {" "}
+                    <span to="https://blog.ethereum.org/2021/05/07/ethereum-for-the-next-billion/">
+                      apply to become a fellow before July 22nd 2022!
```
Comment (wackerow) [inline src/pages/index.js:257]:
```suggestion
                      apply to become a fellow before July 17th 2022!
```

**Review by wackerow — APPROVED** [review]

**Pull request merged**